.github/workflows/publish-swift.yml

```
@@ -28,6 +28,7 @@ jobs:
         env:
           GIT_SSH_COMMAND: "ssh -i /Users/runner/.ssh/id_rsa -o UserKnownHostsFile=/Users/runner/.ssh/known_hosts"
         run: |
+          export SEMANTIC_VERSION=$(echo $GITHUB_REF_NAME | perl -n -e '/.*(\d+\.\d+\.\d+.*)/ && print $1')
```

I think we could do simpler by using a sed script that just removes the first "v" if it exists

```suggestion
          export SEMANTIC_VERSION=$(echo $GITHUB_REF_NAME | sed 's/^v\(.*\)/\1/')
```